int32 val

# success
int32 SUCCESS=1
int32 FAILURE=0
int32 NONE=-1

# general errors
int32 UNKNOWN_COMMAND=100
int32 TIMEOUT=101

# pose errors
int32 MAN_PRE_GRASP_FAILED=200
int32 MAN_GRASP_FAILED=201
int32 MAN_PRE_PLACE_FAILED=202
int32 MAN_PLACE_FAILED=203

# joint target errors
int32 MAN_HOME_FAILED=300

# ee errors
int32 EE_GRASP_FAILED=400
int32 EE_PRE_GRASP_FAILED=401
int32 EE_RELEASE_FAILED=400


# general errors
int32 TRANSFORM_POSE_FAILED=500
